index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="style.css" rel="stylesheet">
+  <title>Product Landing Page</title>
+</head>
+<body>
+  <header id="header" class="container">
+    <img src="https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png" alt="Ori" id="header-img">
+    <nav id="nav-bar">
+      <ul>
+          <li>
+            <a class="nav-link" href="#features">Features</a>
+          </li>
+          <li>
+            <a class="nav-link" href="#how-it-works">How It Works</a>
+          </li>
+          <li>
+            <a class="nav-link" href="#pricing">Pricing</a>
+          </li>
+      </ul>
+    </nav>
+  </header>
+  <!-- <iframe id="video" width="560" height="315" src="https://www.youtube.com/embed/y8Yv4pnO7qc" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe> -->
+  
+ <!--  <section id="newsletter">
+    <form action="https://www.freecodecamp.com/email-submit" id="form">
+      <input name="email" type="email" id="email" placeholder="Enter Email Address">
+      <input type="submit" id="submit">
+    </form>
+  </section> -->
+  <section class=" intro container flexcenter">
+    <div>
+      <h2>Everything You need to Grow your Business</h2>
+    <p>Proton does everything possible to help you grow your business. Attract clients through engaging video content and simplify your business operations</p>
+  <button>Getting Started</button>  
+  </div>
+  </section>
+
+  <section class="container">
+    <img src="corporatewomen.jpg" id="hero-img">
+  </section>
+
+  <footer class=" foot">
+    Built with 💗by Muda
+  </footer>
+  <div  class="flex-container">
```

Comment:
I think you should ensure to avoid too much spacing inside the opening div class, since it's not good practice. read more on HTML and CSS good practice [here](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)